src/hoc/AppSidebar.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line no-unused-vars
+import React, { Fragment } from 'react';
+// eslint-disable-next-line no-unused-vars
+import { Nav, NavItem, NavLink } from 'reactstrap';
+
+// eslint-disable-next-line no-unused-vars
+const AppSidebar = ({ navConfig, ...props }) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)